### ISSUE TITLE
cypress:  prefix snippets to avoid conflicts with other snippets

### ISSUE THREAD

**Issue by digitaldrummerj** [andrew-codes/vscode-snippets#1]
I noticed while trying to use the snippets that I was getting conflicts with other snippets that I have installed unrelated to Cypress.  It would be great if the snippets where prefixed with `cy` or `cy-` so that they don't conflict with other snippets.

**Comment by andrew-codes** [comment]
I agree. I have a couple of things that I have noticed that need to be fixed; including this one. I will spend some time jotting these down as issues and then working to resolve them/getting fixes published. I've been "dealing" with the conflicts for too long and should fix them, so I don't have to deal with them any longer. Stay tuned and thank you for taking the time to open an issue! 👍

**Comment by digitaldrummerj** [comment]
would love to get them before before April 11th if possible so I can pitch it as part of my talk at the CodeStock conference.  

Let me know if you want me to help out with any of it.  I started live streaming this week and I could work on the snippets on my Friday stream this week.

**Comment by andrew-codes** [comment]
@digitaldrummerj I am curious to get your feedback on #3 . There are minor breaking changes. The only breaking change is that `it`, `context` have been moved to another snippet package; `js-testing-syntax`. The problem I was running into is that I work on projects with a combination of either jest and cypress or mocha and cypress. I basically install all three snippets and disable the one not relevant to my workspace (jest or mocha). In order to prevent possible overlap of multiple snippets for the same shortcut when using these snippet packages, I migrated the actual test syntax (`it`, `describe`, `context`, `test`, `suite`, etc.) to this `jest-test-syntax` snippet package.

I tried to prefix snippets intelligently. Cypress snippets are all prefixed with `cy`, and common testing syntax is typically the `tst` prefix with minor deviations.

Jest snippets will eventually become only jest related; matchers, etc. such as `toBe` and `toContain` or `jest.mock` and `jest.fn()`.

**Comment by digitaldrummerj** [comment]
@andrew-codes I would include all of them into the Cypress package so that it is a complete package.  As I don't use jest, I wouldn't think to install the jest-test-syntax  To avoid the overlap, I would just make them cy-it, cy-describe, cy-context, cy-test, cy-suite, etc.  Yes this means you have duplicates in your snippet packages since you are maintaining multiple packages but from a user standpoint, it is only package for me and I get everything that I need for Cypress.

**Comment by digitaldrummerj** [comment]
@andrew-codes how is this coming along?  any help needed?
